client/src/templates/Introduction/components/block.tsx

```diff
@@ -130,9 +134,11 @@ class Block extends Component<BlockProps> {
     });
 
     const blockTitle = t(`intro:${superBlock}.blocks.${blockDashedName}.title`);
+    // the reason we are flating the array, to please TypeScript.
+    // but it isn't needed, because of intro structure as mention in BlockIntos component
     const blockIntroArr = [
       t(`intro:${superBlock}.blocks.${blockDashedName}.intro`)
-    ];
+    ].flat();
```

Comment:
Sorry for the multiple reviews. I am not sure if this is 100%, but it does work:

```typescript
const blockIntroArr: string[] = t<string, never>(
      `intro:${superBlock}.blocks.${blockDashedName}.intro`
    );
```

Reply:
Didn't know that it accepts generics, that makes life easier.

Reply:
I have used `t<string, DefaultTFuncReturn & string[]>` generic type, because I don't like explicts types, feel free to change it, if you think that this isn't worth it. 👍